west.yml

```diff
@@ -96,24 +96,24 @@ manifest:
     # changes.
     - name: mcuboot
       repo-path: sdk-mcuboot
-      revision: 858dd034f6565bcab5ed780e48aa713994ca1b7d
+      revision: v1.7.99-ncs1-rc1
       path: bootloader/mcuboot
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: a21b5e298e5b22340b9dfbd489e8c00483868284
+      revision: v1.5.0-rc1
     - name: trusted-firmware-m
       repo-path: sdk-trusted-firmware-m
       path: modules/tee/tfm
-      revision: 96ef980add272be24442d9140d7c90c80f89e663
+      revision: v1.2.99-ncs1-rc1
     - name: tfm-mcuboot # This is used by the trusted-firmware-m module.
       repo-path: sdk-mcuboot
       path: modules/tee/tfm-mcuboot
       revision: 81d19f017f18f1bc6bea906d96fd45fba3672064
     - name: connectedhomeip
       repo-path: sdk-connectedhomeip
       path: modules/lib/connectedhomeip
-      revision: 3e9c650c91cde7833586acb217fbb6c3ef6a95b0
+      revision: v0.0.99-ncs1-rc1
```

Comment:
Not sure why we didn't use v0.1.99 here, or just v1.5.0?

Reply:
because there is not even a `0.1.0` upstream:
https://github.com/project-chip/connectedhomeip/tags

`0.1.99` would indicate that a `0.1.0` does exists.
I'm fine with tagging with `v1.5.0-rc1` if you prefer.